subprojects/docs/src/docs/userguide/authoring-builds/working_with_files.adoc

```diff
@@ -755,6 +755,20 @@ include::sample[dir="snippets/files/sync/kotlin",files="build.gradle.kts[tags=co
 
 You can also perform the same function in your own tasks with the link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:sync(org.gradle.api.Action)[Project.sync(org.gradle.api.Action)] method.
 
+[[sec:copy_deploy]]
+=== Deploying single files into application servers
+
+When working with application servers, you can use a `Copy` task to deploy the application archive (e.g. a WAR file).
+Since you are deploying a single file, the destination directory of the `Copy` is the whole deployment directory.
+The deployment directory sometimes does contain unreadable files like named pipes, so Gradle may have problems doing up-to-date checks.
+In order to support this use-case, you can use link:{groovyDslPath}/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:doNotTrackDestinationDir()[Copy.doNotTrackDestinationDir()].
+
+.Using Copy to deploy a WAR file
+====
+include::sample[dir="snippets/files/deployWarWithCopy/groovy",files="build.gradle"]
+include::sample[dir="snippets/files/deployWarWithCopy/kotlin",files="build.gradle.kts"]
+====
+
```

Comment:
Is this too specific? Installing your runnable application code to `/usr/share` for example also qualifies here.

Reply:
That is one of the use-cases. Installing single files into directories with many other files is another use-case. `/usr/share/` doesn't seem to be the right example, though, since you'd install into a full sub-directory there, so you don't need untracked necessarily. I suppose `/usr/local/bin` would be the example? I can add this use-case as well.

Reply:
Sorry, yes, I was thinking of `/usr/local/bin`.

Reply:
I added an example for installing an executable.